basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/RootLoggerTestCase.java

```diff
@@ -64,12 +65,18 @@ public void editFilterSpec() throws Exception {
     @Test
     public void editHandlers() throws Exception {
         String handler = "testHandler_" + RandomStringUtils.randomAlphanumeric(5);
+        ModelNodeResult defaultValue = operations.readAttribute(ROOT_LOGGER_ADDRESS, "handlers");
```

Comment:
You should use also `org.wildfly.extras.creaper.core.online.operations.ReadAttributeOption#NOT_INCLUDE_DEFAULTS` to make sure real orig value is returned